frontend/javascripts/oxalis/api/cross_origin_api.js

```diff
@@ -45,6 +45,23 @@ const onMessage = event => {
       }
       break;
     }
+    case "loadPrecomputedMesh": {
+      const segmentId = args[0];
+      const seedPosition = args[1];
+      api.data.loadPrecomputedMesh(segmentId, seedPosition);
+      break;
+    }
+    case "setMeshVisibility": {
+      const segmentId = args[0];
+      const isVisible = args[1];
+      api.data.setMeshVisibility(segmentId, isVisible);
+      break;
+    }
+    case "removeMesh": {
+      const segmentId = args[0];
+      api.data.removeMesh(segmentId);
+      break;
+    }
```

Comment:
I didn't test this part, yet. I think these are the functions which are necessary for the connectome viewer, but I'm not sure. The other functions would also need to return something, but I think we haven't done something like that before :thinking: Maybe we can discuss this in person.

Reply:
> Wouldn't a mesh file need to be activated, first, or is that done automatically?

If there is only one mesh file, this is done automatically, but this needs to be awaited to be robust. Therefore, I also added the corresponding functions now.

> Ideally, we'd implement the connectome viewer use case at the same time to check whether everything is included.

Good point. I opened https://github.com/scalableminds/connectome-viewer/pull/47 :)